File: projetJeu/src/main/java/fr/projetjeu/model/Objet.java
package fr.projetjeu.model;

public class Objet {
private int id;
private String nom;
private boolean typeObjetAlimentaire;
private float prix;
private TypeObjets typeObjets;
private int quantiteInventaire;
private int quantiteBoutique;
private Boutique boutique;
private Inventaire inventaire;

public String getNom() {
	return nom;
}
public void setNom(String nom) {
	this.nom = nom;
}
public int getId() {
	return id;
}
public void setId(int id) {
	this.id = id;
}
public boolean isTypeObjetAlimentaire() {
	return typeObjetAlimentaire;
}
public void setTypeObjetAlimentaire(boolean typeObjetAlimentaire) {
	this.typeObjetAlimentaire = typeObjetAlimentaire;
}
public float getPrix() {
	return prix;
}

public TypeObjets getTypeObjets() {
	return typeObjets;
}
public void setTypeObjets(TypeObjets typeObjets) {
	this.typeObjets = typeObjets;
}
public void setPrix(float prix) {
	this.prix = prix;
}


public int getQuantiteInventaire() {
	return quantiteInventaire;
}
public void setQuantiteInventaire(int quantiteInventaire) {
	this.quantiteInventaire = quantiteInventaire;
}


public int getQuantiteBoutique() {
	return quantiteBoutique;
}
public void setQuantiteBoutique(int quantiteBoutique) {
	this.quantiteBoutique = quantiteBoutique;
}


public Boutique getBoutique() {
	return boutique;
}
public void setBoutique(Boutique boutique) {
	this.boutique = boutique;
}


public Inventaire getInventaire() {
	return inventaire;
}
public void setInventaire(Inventaire inventaire) {
	this.inventaire = inventaire;
}


public String toString() {
	return this.nom +"--Prix="+ this.prix+"--Qte dans boutique="+this.quantiteBoutique +"--Qte votre inventaire="+this.quantiteInventaire;
}
}

File: ProjetAngular/src/app/pagesweb/model/objet.ts
export class Objet {
  public get typeObjets(): string|undefined {
    return this._typeObjets;
  }
  public set typeObjets(value: string|undefined) {
    this._typeObjets = value;
  }
  public get prix(): number |undefined{
    return this._prix;
  }
  public set prix(value: number|undefined) {
    this._prix = value;
  }
  public get typeObjetAlimetaire(): boolean |undefined{
    return this._typeObjetAlimetaire;
  }
  public set typeObjetAlimetaire(value: boolean|undefined) {
    this._typeObjetAlimetaire = value;
  }
  public get nom(): string |undefined{
    return this._nom;
  }
  public set nom(value: string|undefined) {
    this._nom = value;
  }
  public get id(): number |undefined{
    return this._id;
  }
  public set id(value: number|undefined) {
    this._id = value;
  }

  constructor(
      private _id?: number,
      private _nom?: string,
      private _typeObjetAlimetaire?: boolean,
      private _prix?: number,
      private _typeObjets?: string
  ){}
}
